src/static/js/arbitrum_helpers.js

```diff
@@ -259,6 +260,40 @@ async function getArbitrumDlpPool(App, dlpPool, tokenAddress, originTokenAddress
   }
 }
 
+async function getArbitrumCurvePool(App, tokenAddress, stakingAddress) {
+  const crv = new ethers.Contract(tokenAddress, CURVE_ABI, App.provider);
+  let minterAddress = tokenAddress;
+  try{
+    minterAddress = await crv.minter();
+  } catch(err) {
+  }
+  const minter = new ethers.Contract(minterAddress, MINTER_ABI, App.provider);
```

Comment:
we've updated arbitrum to use ethcall, so these could all be done in 1 call now. we can get it updated Monday probably